.gitignore

```diff
@@ -141,3 +141,6 @@ output/*
 _start
 
 src/.idea/
+
+#JetBrains Rider
```

Comment:
Nit: `# Jetbrains` since it applies to everything (and consistent with other comments).